ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -161,6 +170,31 @@ func (scbuilder *SiteConfigBuilder) getClusterCR(clusterId int, siteConfigTemp S
 	return mapIntf, nil
 }
 
+func (scbuilder *SiteConfigBuilder) getBiosConfigMapCR(clusterId int, siteConfigTemp SiteConfig, nodeId int) (interface{}, error) {
```

Comment:
Minor nit. This method should be "buildBiosConfigMapCR" (or similar). Using "get" for the name makes it sound like this is simply looking it up and returning it, but this method contructs a new ConfigMap and returns it.

Reply:
Agree.